packages/canvas-high-contrast-theme/package.json

```diff
@@ -8,7 +8,8 @@
     ".": {
       "require": "./lib/index.js",
       "default": "./es/index.js"
-    }
+    },
+    "./package.json": "./package.json"
```

Comment:
This (and a few similar ones) are tricky. They cause an exception during build, but the build passes. Without adding these the `build:tokens` script breaks (this generates themes)